cfgov/v1/migrations/0032_cfgovimage_cfgovrendition.py

```diff
@@ -14,16 +14,16 @@ def migrate_images_data(apps, schema_editor):
     UserModel = apps.get_model(settings.AUTH_USER_MODEL)
```

Comment:
after this update, we really should squash these migrations. All this stuff is 100% unnecessary